lambda/requirements.txt

```diff
@@ -0,0 +1,3 @@
+google-api-python-client 
+google-auth-httplib2 
+google-auth-oauthlib
```

Comment:
Can pipenv generate this file for us as part of the `build-google-groups-lambda` make target?

Reply:
bump

Reply:
You're right it can - and now it is